website/docs/d/endpoint_v1.html.markdown

```diff
@@ -0,0 +1,80 @@
+---
+subcategory: "core/v1"
+layout: "kubernetes"
+page_title: "Kubernetes: kubernetes_endpoints_v1"
+description: |-
+An Endpoints resource is an abstraction, linked to a Service, which defines the list of endpoints that actually implement the service.
+---
+
+# kubernetes_endpoints_v1
+
+An Endpoints resource is an abstraction, linked to a Service, which defines the list of endpoints that actually implement the service.
+
+
+## Example Usage
+
+```hcl
+data "kubernetes_endpoints" "api_endpoints" {
```

Comment:
Could you please change data source name from `kubernetes_endpoints` to `kubernetes_endpoints_v1`?